lib/migration.js

```diff
@@ -122,7 +122,7 @@ Migration.prototype.sqlFileLoaderTemplate = function() {
     '};',
     '',
     'exports.up = function(db) {',
-    '  var filePath = path.join(__dirname + \'/sqls/' + this.name.replace(
+    '  var filePath = path.join(__dirname, \'/sqls/' + this.name.replace(
```

Comment:
I know this comes a bit late, but if you're that pernickety, while you're obviously right, path.join should be used correctly anyway ;)

So instead of `\'/sqls/' + this.name.replace(` placing `this.name.replace` as another argument as path.join accepts an unlimited amount of parameters.
If you change this, then I'm going to merge this one.